main.js

```diff
@@ -52,29 +52,51 @@ var pastActivities = [];
 
 //Add event listeners below
 categoryBoxWrapper.addEventListener('click', findCategory);
+
 startActivityButton.addEventListener('click', submitForm);
+
 startTimerButton.addEventListener('click', function() {
   startTimerButton.classList.add('cannot-click');
   currentActivity.startTimer();
 });
+
 logActivityButton.addEventListener('click', logActivityEvents);
+
 createNewActivityButton.addEventListener('click', returnToActivityForm);
 
+// Add functions below
 window.onload = function() {
```

Comment:
Nice work creating the appropriate logic in the windows load function. That was tricky.

Reply:
Thanks! I just realized we still have a little cleanup to do here, which we can address when we do the cleaning pass tomorrow. We'll need to get rid of the `console.log` statements and the alert in this `onload` function.